src/main.ts

```diff
@@ -114,6 +198,32 @@ async function main() {
 		return await handler.fetchPayoutInfo(hash, address);
 	});
 
+	// GET staking information for an address.
+	//
+	// Paths:
+	// - `address`: The _Controller_ address for staking.
+	// - (Optional) `number`: Block number at which to query. If not provided, queries finalized head.
+	//
+	// Returns:
+	// - `at`: Block number and hash at which the call was made.
+	// - `stash`: The stash account whose balance is actually locked and at stake.
```

Comment:
If we want to address my above comment we could add the vice versa here; Stash could be the same as controller